lib/Elastica/Query/Ids.php

```diff
@@ -77,7 +77,7 @@ public function setType($type)
             return $this;
         }
 
-        $this->_params['type'] = $type;
+        $this->_params['type'] = (array)$type;
```

Comment:
@ReenExe I think you just discovered a bug. I missed `addType`. Unfortunately we don't have any unit test for this. It seems we have tests for `setType` so the string part seems to work as expected. I'm curious of addType would even properly work with elasticsearch.